pkg/node/node.go

```diff
@@ -105,6 +113,25 @@ func (n Node) CordonAndDrain(nodeName string) error {
 	if err != nil {
 		return err
 	}
+	// Emit events for all pods that will be evicted
+	if recorder != nil {
```

Comment:
I know if `nthConfig.EmitKubernetesEvents == false` then `recorder == nil` , but I think we should pass the entire `nthConfig` in. One reason is for readability (I suspect upstream folks will want that) and another reason is we could also stamp `nthConfig.KubernetesEventsExtraAnnotations` into the Pod events.

Reply:
Or maybe just these two params would suffice, you get the point.